src/App/component/ProjectAvatar.jsx

```diff
@@ -111,73 +133,132 @@ function ProjectAvatar(props) {
       })
   }
 
+  const renameProject = async () => {
+    const projectId = props.project.projectId
+    try {
+      await Axios.patch(`http://localhost:9100/pvs-api/project?name=${projectName}&projectId=${projectId}`)
```

Comment:
```suggestion
      await Axios.patch(`http://localhost:9100/pvs-api/project/name?name=${projectName}&projectId=${projectId}`)
```

Suggest this change because of the change of the PR [Feat: Add functions of rename project](https://github.com/SE-8-2021/backend/pull/43)

Reply:
fix: change api path to fit with backend

Reply:
Addressed in f2906c2.